From a65eb702a160882ba7729efc8746800145cc0353 Mon Sep 17 00:00:00 2001
From: Andrew Boie <andrew.p.boie@intel.com>
Date: Tue, 27 Nov 2012 11:43:30 -0800
Subject: MUST_REVERT: Use host ccache if present

ccache works much better if a 3.x version is in use. If the build
host has ccache installed, use that instead of the prebuilt binary
which is still version 2.4.

This can be reverted once Google updates the prebuilt ccache to a
3.x version.

Change-Id: I829c6aa9b1b332753e7aea7334cab3507191fb74
---
 core/combo/select.mk |    6 +++++-
 1 files changed, 5 insertions(+), 1 deletions(-)

diff --git a/core/combo/select.mk b/core/combo/select.mk
index b303bad..d9a64f8 100644
--- a/core/combo/select.mk
+++ b/core/combo/select.mk
@@ -83,7 +83,11 @@ ifneq ($(USE_CCACHE),)
   ifeq ($(HOST_OS)-$(BUILD_OS),windows-linux)
     CCACHE_HOST_TAG := linux-$(BUILD_ARCH)
   endif
-  CCACHE_BIN := prebuilts/misc/$(CCACHE_HOST_TAG)/ccache/ccache
+  CCACHE_BIN := $(shell which ccache)
+  ifndef CCACHE_BIN
+    CCACHE_BIN := prebuilts/misc/$(CCACHE_HOST_TAG)/ccache/ccache
+    $(warning Using prebuilt $(shell $(CCACHE_BIN) -V | sed q); install a 3.x ccache on your workstation for better performance)
+  endif
   # Check that the executable is here.
   CCACHE_BIN := $(strip $(wildcard $(CCACHE_BIN)))
   ifdef CCACHE_BIN
-- 
1.7.4.1

